.github/workflows/ci-linux.yml

```diff
@@ -25,5 +32,5 @@ jobs:
     - name: Upload Flatpak
       uses: actions/upload-artifact@v3
       with:
-        name: obliteration-linux
+        name: obliteration-linux64
```

Comment:
Now we are going to support both AMD64 and ARM64 so this name is ambiguous. We should rename it to something like `linux-x86_64` for Linux and `windows-x64` for Windows.